client/actions/planning/ui.js

```diff
@@ -558,6 +558,36 @@ const addNewCoverageToPlanning = (coverageType, item) => (
     })))
 );
 
+const openCancelCoverageModal = (planning, coverage, index, onSubmit, onCancel) => (
+    (dispatch, getState) =>
+        dispatch(showModal({
+            modalType: MODALS.ITEM_ACTIONS_MODAL,
+            modalProps: {
+                original: planning,
+                actionType: COVERAGES.ITEM_ACTIONS.CANCEL_COVERAGE.label,
+                coverage: coverage,
+                index: index,
+                onSubmit: onSubmit,
+                onCancel: onCancel,
+            },
+        }))
+);
+
+const cancelCoverage = (original, updatedCoverage, index) => (
```

Comment:
This action should be in `client/actions/planning/api.js` as it's an api call.

Reply:
Subsequent to the api call, there is UI notification and further processing of returned data. Similar examples in the same file: addCoverageToWorkflow, removeAssignment, spike, etc.

Reply:
Fair enough